cypress/integration/admin/pageBuilder/page/publishPage.spec.js

```diff
@@ -1,97 +1,113 @@
-//             cy.findByTestId("pb.editor.header.publish.button").click();
-//             cy.findByTestId("pb-editor-publish-confirmation-dialog").within(() => {
-//                 cy.findByTestId("confirmationdialog-confirm-action").click();
-//             });
-//         });
-//
-//         it.skip(`Step 4: Check updated page title in page preview`, () => {
-//             return cy.pbListPages({ limit: 1, search: { query: pageTitle2 } }).then(([page]) => {
-//                 const { path } = page;
-//                 cy.visit(`${Cypress.env("WEBSITE_URL")}${path}`);
-//
-//                 cy.title().should("contain", pageTitle2);
-//             });
-//         });
-//
-//         it.skip(`Step 5: Delete page immediately`, () => {
-//             cy.visit("/page-builder/pages");
-//             return cy.pbListPages({ limit: 1, search: { query: pageTitle2 } }).then(([page]) => {
-//                 const { id } = page;
-//                 // Delete page by deleting first revision
-//                 cy.pbDeletePage({ id: `${id.slice(0, -1)}1` });
-//             });
-//         });
-//     }
-// );
+import uniqid from "uniqid";
+
+context(
+    "Should be able to publish and re-publish a page, and see changes on the public site",
+    () => {
+        let id;
+        const pageTitle1 = `Test published page 1`;
+        const pageTitle2 = `Test published page 2`;
+
+        beforeEach(() => {
+            id = uniqid();
+            cy.login();
+            cy.pbDeleteAllPages();
+            cy.pbCreatePage({ category: "static" }).then(page => {
+                cy.pbUpdatePage({
+                    id: page.id,
+                    data: {
+                        category: "static",
+                        path: `/page-${id}`,
+                        title: pageTitle1
+                    }
+                });
+                cy.pbPublishPage({ id: page.id });
+            });
+        });
+
+        describe("When Publishing new page", () => {
+            it("Step 1: Check page title in preview", () => {
+                cy.visit("/page-builder/pages");
+
+                // Check title on the page preview drawer.
+                cy.findByText(`${pageTitle1}`, { timeout: 15000 }).click({ force: true });
+                // Check whether we oppened a page preview drawer.
+                cy.url().should("include", "&id");
+                // Checking if title is correct.
+                cy.findByTestId("page-details-page-title").contains(`${pageTitle1}`);
+
+                // As we don't have a button to close page preview drawer,
+                // we need to simulate click outside of the drawer to close it.
+                cy.get("body").click("topLeft");
+                // Check if we closed page preview drawer.
+                cy.url().should("not.contain", "&id");
+
+                // Check title on the public site.
+                cy.pbListPages({ limit: 1, search: { query: pageTitle1 } }).then(pages => {
```

Comment:
Let's fetch a page by creating a new `cy.pbGetPage` command. We can do better here.

Reply:
@adrians5j I have created this command https://github.com/webiny/webiny-js/pull/3535/files#diff-427581abae631e9e0aabefea94ecd56b9f9b037b82796c333726a4bba2da08f5R3